core/src/components/manager_controller_component.cpp

```diff
@@ -99,20 +98,39 @@ formats::json::Value ManagerControllerComponent::ExtendStatistics(
     const utils::statistics::StatisticsRequest& /*request*/) {
   formats::json::ValueBuilder engine_data(formats::json::Type::kObject);
 
-  formats::json::ValueBuilder json_task_processors(
-      formats::json::Type::kObject);
-  for (const auto& [name, task_processor] :
-       components_manager_.GetTaskProcessorsMap()) {
-    json_task_processors[name] = GetTaskProcessorStats(*task_processor);
+  // task processors
+  {
+    formats::json::ValueBuilder json_task_processors(
+        formats::json::Type::kObject);
+    for (const auto& [name, task_processor] :
+         components_manager_.GetTaskProcessorsMap()) {
+      json_task_processors[name] = GetTaskProcessorStats(*task_processor);
+    }
+    utils::statistics::SolomonChildrenAreLabelValues(json_task_processors,
+                                                     "task_processor");
+    utils::statistics::SolomonSkip(json_task_processors);
+    engine_data["task-processors"]["by-name"] = std::move(json_task_processors);
+  }
+
+  // ev-threads
+  {
+    formats::json::ValueBuilder json_ev_threads{formats::json::Type::kObject};
+
+    const auto& pools_ptr = components_manager_.GetTaskProcessorPools();
+    auto& ev_thread_pool = pools_ptr->EventThreadPool();
+    for (auto* thread : ev_thread_pool.NextThreads(ev_thread_pool.GetSize())) {
+      json_ev_threads[thread->GetName()] = thread->GetCurrentLoadPct();
```

Comment:
maybe it's better to group threads by pool name and leave thread number as a variable part
like 'event.1'

Reply:
Well, there's no such thing as `thread number` currently, and i didn't want to introduce it upfront.
Definitely doable, so i can if you insist.